### PR TITLE
add a dialog if not on coh3stats.com (such as dev site) https://github.com/cohstats/coh3-stats/issues/173

### PR DESCRIPTION
**Pull request by rysllvn**
https://github.com/cohstats/coh3-stats/issues/173

not sure if it's in the best spot, let me know. It's in `_app.tsx`

### CODE REVIEW

File: components/dev-site-notification.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Anchor, Dialog, Text } from "@mantine/core";
+import { useDisclosure } from "@mantine/hooks";
+import { useEffect } from "react";
+
+const DevSiteNotification = () => {
+  const [opened, { open, close }] = useDisclosure(false);
+  useEffect(() => {
+    if (window.location.hostname !== "coh3stats.com") {
```
Comment (petrvecera) [inline components/dev-site-notification.tsx:8]: We can use this function https://github.com/cohstats/coh3-stats/blob/master/config.ts#L31

Comment (petrvecera) [reply components/dev-site-notification.tsx:8]: Also we could use that function in in the app directly in the rendering:
```
{isDevEnv() && <DevSiteNotification />} 

```

So it's not even rendered on the main page! :)

Comment (rysllvn) [reply components/dev-site-notification.tsx]: done

**Review by petrvecera — APPROVED** [review]
Love it 👍 ❤️

**Pull request merged**